### ISSUE TITLE
Paper Metadata Dataframe

### ISSUE THREAD

**Issue by jermnelson** [sul-dlss-labs/spoc#47]
Moving into it's own ticket from #38, we'll want to use normalize the metadata for the various institutions that are currently in the Google Docs [spreadsheet](https://docs.google.com/spreadsheets/d/1EjCjMBLPdELkdR-XSz31gv8jZwBOCwCZEG0vNlLj57I/edit#gid=1830310682).

From the converted TEI XML in `SUL AI 2020-2021/Project - Species Occurrences/papers_tei` and the Google metadata [Spreadsheet](https://docs.google.com/spreadsheets/d/1EjCjMBLPdELkdR-XSz31gv8jZwBOCwCZEG0vNlLj57I/edit?usp=drive_fs&ouid=102548702195264613391), create a _papers_ Dataframe with the following columns:

  1. **XML filename**
  2. **Title**
  3. **Institution**
  4. **Author**
  5. **Secondary Author**
  6. **Abstract**
  7. **Year**

**Comment by amandawhitmire** [comment]
@jermnelson - I can normalize metadata in the spreadsheet if it helps with your process to generate the dataframe. I was thinking of creating a standalone spreadsheet that pulls together all of the institutional records into one sheet. Is that useful?

**Comment by jermnelson** [comment]
> @jermnelson - I can normalize metadata in the spreadsheet if it helps with your process to generate the dataframe. I was thinking of creating a standalone spreadsheet that pulls together all of the institutional records into one sheet. Is that useful?

Yes, @amandawhitmire  creating a standalone spreadsheet would be very helpful. There is a very good possibility that the one sheet could be read into a dataframe that we could directly in the app.
